### ISSUE TITLE
Spellcheck while typing

### ISSUE THREAD

**Issue by jorilallo** [ianstormtaylor/slate#1804]
#### Do you want to request a _feature_ or report a _bug_?

- Bug

#### What's the current behavior?

![kapture 2018-04-30 at 13 42 36](https://user-images.githubusercontent.com/31465/39449295-81f672d0-4c7c-11e8-9250-61d7063001be.gif)

Spellchecking checks while typing.

#### What's the expected behavior?

With native input fields, spellchecking seems to be applied only after space or other indication that the user has stopped typing the word.

Not sure if this is even possible to fix, curious if anyone has addressed this.

**Comment by zhujinxuan** [comment]
I do not know.  I think it is the strange behavior of Chrome.  We can event reproduce it in `<textarea/>`

**Comment by jorilallo** [comment]
@zhujinxuan I'm not able to, it will only do the red underline after space is applied, not while typing the word

**Comment by steobrien** [comment]
I think this is a duplicate of https://github.com/ianstormtaylor/slate/issues/1598.

**Comment by zhujinxuan** [comment]
@steobrien @jorilallo  I find the solution.  The idea is to prevent the react-rendering in necessary cases.  

I will work on that next week.

**Comment by dmitrizzle** [comment]
FWIW spell check with Slate is already a lot better than what Medium.com has (M basically does not work at all). Looking forward to this fix 💪

**Comment by Slapbox** [comment]
@zhujinxuan did you make progress on this issue?

**Comment by zhujinxuan** [comment]
@Slapbox I am not working on this recently.  It may include a lot of shouldComponentUpdate changes.  I am working on something else, and perhaps switch back to this issue in Nov

**Comment by ianstormtaylor** [comment]
Closing in favor of #1598, as @steobrien reported.
